### PR TITLE
Create; mixin and helper for text-stretch

### PR DESCRIPTION
**Pull request by chrisopedia**
Why is this pull request necessary:

1. common pattern seen across designs

Changes proposed in this pull request:

- create mixin
- create helper
- update example

### CODE REVIEW

File: styles/helpers/typography.css [hunk 1]
```diff
@@ -30,6 +30,10 @@ body {
   text-align: right;
 }
 
+.text-stretch {
+  @mixin text-stretch 2px;
```
Comment (jgallen23) [inline styles/helpers/typography.css]: how consistent is 2px all the time?  should this be project specifc?

Comment (chrisopedia) [inline styles/helpers/typography.css]: Could be. So far from the few designs I've seen it looks like this is the norm.

Comment (jgallen23) [inline styles/helpers/typography.css]: feels like the 2px should be a var driven by the config and then sites could override the yaml var if they wanted to tweak it

**Comment by jgallen23** [comment]
if you are going to do text stretch, just add this to fontStyles:

`text-stretch: 
  letter-spacing: 2px
`
and that'll auto create a helper class.  Make sure you are updating the tests also.  run `npm test`

File: styles/helpers/typography.css [hunk 2]
```diff
@@ -30,6 +30,10 @@ body {
   text-align: right;
 }
 
+.text-stretch {
```
Comment (jgallen23) [inline styles/helpers/typography.css]: don't need this anymore

File: styles/mixins/text-stretch.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module.exports = function ( config ) {
```
Comment (jgallen23) [inline styles/mixins/text-stretch.js]: you can kill this

File: test/expectedOutputs/mixins/font-style.js [hunk 4]
```diff
@@ -78,5 +78,11 @@ module.exports={
       "font-size": "12px",
       "line-height": "22px"
     }
+  },
+  ".text-stretch": {
+    "letter-spacing": "2px",
+    "@media (max-width: 767px)": {
```
Comment (jgallen23) [inline test/expectedOutputs/mixins/font-style.js]: hmm, we should put a check in here so it doesn't output if it's not there

Comment (jgallen23) [inline test/expectedOutputs/mixins/font-style.js]: I'll add a ticket for it

**Pull request merged**